modules/ROOT/pages/promote-api.adoc

```diff
@@ -52,20 +52,12 @@ To promote an API:
 . Click *Next: Provide Authentication*.
 . On the *Provide authentication* page, select the type of authentication on the GET endpoints for the API that you want to add, allowing Anypoint DataGraph to make requests to your API when queried:
 +
---
 * *No Auth*: Use if your API is public.
 * *Basic Auth*: Use if you authenticate to your API using a username and password.
+* *Pass-through*: Use if you authenticate to your API by passing authorization headers.
 * *Client ID enforcement via headers*: Use if you authenticate to your API passing `client_id` and `client_secret` headers.
 * *Client ID enforcement via query parameters*: Use if you authenticate to your API passing `client_id` and `client_secret` query params.
 * *Custom*: Use if you authenticate to your API using custom header parameters and values.
---
-+
-[IMPORTANT]
---
-Any other authentication methods to access your API, such as custom OAuth implementations or short-lived token methods, are not supported.
```

Comment:
Some contextual information for you @IsaacEldridge ... we are planning to implement OAuth support by End-Q4 (To be confirmed this week)

Reply:
got it!